### PR TITLE
Closes #909 - SHR u claim must include port number, if supplied

### LINKED ISSUE #909: AT/PoP - Host 'u' claim must include port number, if supplied
See [draft-ietf-oauth-signed-http-request-03#section-3](https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3)

>   u  :: OPTIONAL.  The HTTP URL host component as a JSON string.  **This MAY
      include the port separated from the host by a colon in host:port
      format.**

### PR DESCRIPTION
**Pull request by iambmelt**
See #909

In this PR:
- Formatting tweaks for claims
- Include port number in `u`
- Adds new unit tests

### CODE REVIEW

**Comment by shahzaibj** [comment]
@iambmelt Looks like all the new tests are failing

**Comment by iambmelt** [comment]
@shahzaibj Yeah, I'll need to investigate why these don't work on an emulator, but pass on a real device thx

File: common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java [hunk 1]
```diff
@@ -756,7 +778,7 @@ private static KeyPair getKeyPairForEntry(@NonNull final KeyStore.Entry entry) {
      */
     private static RSAKey getRsaKeyForKeyPair(@NonNull final KeyPair keyPair) {
         return new RSAKey.Builder((RSAPublicKey) keyPair.getPublic())
-                .privateKey(keyPair.getPrivate())
+                //.privateKey(keyPair.getPrivate())
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java:781]: Remove commented out code

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java]: 1bf8003b6b5e4ebc1a2a93725b548762c5e7b3a4

**Pull request merged**